srcpkgs/python3-utils/template

```diff
@@ -1,19 +1,21 @@
 # Template file for 'python3-utils'
 pkgname=python3-utils
-version=3.1.0
-revision=3
+version=3.5.2
+revision=1
 build_style=python3-module
-# skip coverage and mypy tests https://github.com/WoLpH/python-utils/issues/29
-make_check_args="-o addopts=--doctest-modules"
 hostmakedepends="python3-setuptools"
 depends="python3"
 checkdepends="python3-pytest python3-pytest-asyncio"
 short_desc="Convenient utilities not included with the standard Python3 install"
 maintainer="Karl Nilsson <karl.robert.nilsson@gmail.com>"
 license="BSD-3-Clause"
-homepage="https://github.com/WoLpH/python-utils"
-distfiles="https://github.com/WoLpH/python-utils/archive/v${version}.tar.gz"
-checksum=6f20e784af79dd0f2c330c88496b3d08fa29e27bcfba398ba78f73d8ae85f670
+homepage="https://python-utils.readthedocs.org/en/latest/"
+distfiles="https://github.com/wolph/python-utils/archive/refs/tags/v${version}.tar.gz"
+checksum=390679454ce9d9f33bff932b17d75648a9cfddc23e771cf6c4c37a1274b3f66e
+
+pre_check() {
+	rm pytest.ini _python_utils_tests/test_logger.py # requires unpackaged loguru
+}
```

Comment:
check can be ran multiple times, this will be failing on the second run, move it to post_extract which runs only once

Reply:
but abby told me the other day that the tests aren't meant to run multiple times without running clean first; that's why I have been avoiding -f with rm

Reply:
https://github.com/void-linux/void-packages/pull/42294#discussion_r1108062163

Reply:
hmm, I sometimes run them multiple times without cleaning and without any issues